pkg/agent/route/route_linux.go

```diff
@@ -772,3 +775,124 @@ func (c *Client) DeleteSNATRule(mark uint32) error {
 	snatIP := value.(net.IP)
 	return c.ipt.DeleteRule(iptables.NATTable, antreaPostRoutingChain, c.snatRuleSpec(snatIP, mark))
 }
+
+func (c *Client) AddNodePortConfig(nodePortIPMap map[int][]net.IP) error {
+	gatewayIfIndex := getIndexByName(c.nodeConfig.GatewayConfig.Name)
+	// Clean left TC config.
+	qdiscsClean(nodePortIPMap, gatewayIfIndex)
+
+	// Add qdisc to interfaces excluding antrea-gw0.
```

Comment:
What you mean by "excluding antrea-gw0" here? BTW, better to use "gateway interface" rather than "antrea-gw0", as technically the gateway interface name can be changed.

Reply:
Agreed, that's more rigorous.

Reply:
Could you clarify on my question of what does "excluding antrea-gw0" mean here? I did not figure out from code.

Reply:
Because the IP address of Antrea gateway can't be accessed from remote, in my opinion, the available IP addresses of NodePort Service should not include the IP address of Antrea gateway(default is `antrea-gw0`).

Reply:
I feel you might need to rephase the comment here (or should it be at line 796). I could not associate the comment with the followed "for" block.